api/sys_init.go

```diff
@@ -31,24 +31,31 @@ func (c *Sys) Init(opts *InitRequest) (*InitResponse, error) {
 }
 
 type InitRequest struct {
-	SecretShares      int      `json:"secret_shares"`
-	SecretThreshold   int      `json:"secret_threshold"`
-	StoredShares      int      `json:"stored_shares"`
-	PGPKeys           []string `json:"pgp_keys"`
-	RecoveryShares    int      `json:"recovery_shares"`
-	RecoveryThreshold int      `json:"recovery_threshold"`
-	RecoveryPGPKeys   []string `json:"recovery_pgp_keys"`
-	RootTokenPGPKey   string   `json:"root_token_pgp_key"`
+	SecretShares         int      `json:"secret_shares"`
+	SecretThreshold      int      `json:"secret_threshold"`
+	StoredShares         int      `json:"stored_shares"`
+	EnableKeyIdentifiers bool     `json:"enable_key_identifiers"`
```

Comment:
Any reason to add yet another option? Why not just default to on? I don't see a good reason to disable it.